var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -361,6 +342,21 @@ class Lammps(CMakePackage, CudaPackage, ROCmPackage):
         deprecated=True,
     )
 
+    stable_versions = {
+        "20230802",
+        "20220623.4",
+        "20220623.3",
+        "20220623.2",
+        "20220623.1",
+        "20220623",
+        "20210929.3",
+        "20210929.2",
+        "20210929.1",
+        "20210929",
+        "20201029",
+        "20200303",
+    }
+
```

Comment:
Why is this a dictionary (versus a list)?

Reply:
This is a [set](https://docs.python.org/3/library/stdtypes.html#set-types-set-frozenset).

Reply:
🤦

Reply:
I really need to stop trying to do too many things at the same time.